integration_test_project/seeds/monthly_spend_fixture.csv

```diff
@@ -1,6 +1,6 @@
 MONTH,SERVICE,SPEND
-2022-04-01,Storage,1.136266305
-2022-04-01,Compute,912.772222176
-2022-04-01,Adj For Incl Cloud Services,-74.921528302
-2022-04-01,Cloud Services,74.921528302
-2022-04-01,Automatic Clustering,0.051113362
+2022-04-01,Storage,1.099019393
+2022-04-01,Compute,885.384444396
+2022-04-01,Adj For Incl Cloud Services,-72.672409968
+2022-04-01,Cloud Services,72.672409968
+2022-04-01,Automatic Clustering,0.049169466
```

Comment:
#95 caused the integration tests to fail. The reason for this failure is that the daily spend model now filters out incomplete dates: https://github.com/get-select/dbt-snowflake-monitoring/pull/95/files#diff-670883b931e3764f94c53438f1378239b044c987c2fa15f55e09f8f51e03611dR64-R66

I had to add this change in because we previously relied on an `inner join` to `daily_rates` to only output complete days. in #95 that was switched to a left join, so to ensure consistency we need to restrict incomplete days from appearing in the output by calculating a stop threshold.

the mock data has data for the month of april, but some of the inputs only have data up until the 29th.

rather than fill the missing inputs, i updated the expected output